### DISCUSSION TITLE
How to get Student benefits?

### DISCUSSION THREAD

**Discussion by TungusSs** [community/community#17728] [GitHub Education]
Hello! I decided to get Student benefits and start learning programming to the fullest. My college is on the list, but when I moved on, I saw that you need my student ID or something that confirms that I really am a student. The problem is that I can’t upload a photo, because as it says, I need to turn on the camera on my computer and show my student ID there in order to apply. But I don’t have a camera or a laptop with a camera, what should I do?
Best regards.

**Comment by suprstarrd** [comment]
Do you have a phone? If you do, then you can probably apply through your phone and upload a picture that way as most phones do have cameras.

**Reply by TungusSs** [reply]
I thought so too and decided to do it with my phone. But, unfortunately, everything is handwritten on my student ID, so after sending the request, I immediately received a refusal, apparently from a bot that checks the photos automatically and it seems he could not understand what was written there

**Comment by mlarasusan** [comment]
Hello @TungusSs 👋 


👉 Check [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) to learn how to get the GitHub Student Developer Pack without a student ID
👉 Also, check the reviews of these two students: [here](https://github.com/github-community/community/discussions/31#discussioncomment-2561585) & [here](https://github.com/orgs/github-community/discussions/16)
 👉 And here is the [link](https://support.github.com/contact/education) to contact our support team [✨](https://emojipedia.org/sparkles/)

**Comment by elisemoe** [comment]
Hi @TungusSs  👋  Would you be able to apply from a phone camera?

Sometimes, for a number of reasons, our system needs additional levels of verification and asks that you take an image rather than upload one. Other suggestions might be to try applying while you are on campus, if able to.
